packages/kbn-openapi-bundler/src/__test__/circular/circular_ref_specs/expected.yaml

```diff
@@ -0,0 +1,52 @@
+openapi: 3.0.3
+info:
+  title: Bundled OpenAPI specs
+  version: '2023-10-31'
+servers:
+  - url: 'http://{kibana_host}:{port}'
+    variables:
+      kibana_host:
+        default: 'localhost'
+      port:
+        default: '5601'
+security:
+  - BasicAuth: []
+paths:
+  /api/some_api:
+    get:
+      operationId: TestEndpointGet
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CircularTestSchema'
+    post:
+      operationId: TestEndpointPost
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CircularTestSchema'
+components:
+  securitySchemes:
+    BasicAuth:
+      scheme: basic
+      type: http
+  schemas:
+    CircularTestSchema:
+      type: string
+      data:
+        items:
+          $ref: '#/components/schemas/AnotherCircularTestSchema'
```

Comment:
Sure it's not valid. Technically it doesn't have to be valid since OAS bundler doesn't check OAS validity.

Anyway I fixed all invalid OpenAPI types.